ONSdigital/eq-terraform
# Split VPC creation from survey runner infrastructure creation

### What is the context of this PR?

The survey runner VPC and tools subnet creation have been moved into a separate terraform run. This has been done for two reasons:
- Associating a VPN connection to a VPC is a manual process and needs to be done each time we want to re-build the survey runner. Separating out the VPC creation means we can automate the creation and destruction of the rest of the survey runner infrastructure.
- The hand crafted Jenkins box used to create the survey runner infrastructure can be left running in the VPC, removing the need to snapshot and re-create on every refresh of the survey runner infrastructure.
### How to review
- Create a new survey runner environment by running the survey-runner.sh script - `./survey_runner.sh apply`
- Verify that you can deploy the survey runner application and it works
- Destroy the survey runner environment by running the survey-runner.sh script - `./survey_runner.sh destroy`
### Who worked on the PR

@ajmaddaford
